warp/tests/tile/test_tile_load.py

```
@@ -32,6 +32,118 @@
 TILE_NPOT_3D = wp.constant(7)
 
 
+@wp.kernel(enable_backward=False)
+def tile_negative_offset_1d_register_kernel(
+    input: wp.array1d[float],
```

Ruff (0.16.1)

[error] Function argument `input` is shadowing a Python builtin

(A002)